examples/chip-tool/README.md

```diff
@@ -12,48 +12,68 @@ An example application that uses CHIP to send messages to a CHIP server.
 
 ## Building the Example Application
 
-Building the example application is quite straightforward.
+Building the example application is quite straightforward. It can either be done
+as part of an overall "build everything" build:
 
-### gn
+```
+./gn_build.sh
+```
+
+which puts the binary at `out/debug/standalone/chip-tool` or directly via:
 
 ```
-cd examples/chip-tool
-git submodule update --init
-source third_party/connectedhomeip/scripts/activate.sh
-gn gen out/debug
-ninja -C out/debug
+scripts/examples/gn_build_example.sh examples/chip-tool SOME-PATH/
 ```
 
--   After the application is built, it can be found in the build directory as
-    `out/debug/chip-tool`
+which puts the binary at `SOME-PATH/chip-tool`.
 
-## Using the Client to Pair a device
+## Using the Client to commission a device
 
-In order to send commands to a device, it must be paired with the client.
+In order to send commands to a device, it must be commissioned with the client.
+chip-tool currently only supports commissioning and remembering one device at a
+time. The configuration state is stored in `/tmp/chip_tool_config.ini`; deleting
+this and other `.ini` files in `/tmp` can sometimes resolve issues due to stale
+configuration.
 
-#### Pair a device
+#### Commission a device
 
-To initiate a client pairing request to a device, run the built executable and
-choose the pairing mode.
+To initiate a client commissioning request to a device, run the built executable
+and choose the pairing mode.
 
-##### Pair a device configured to bypass Rendezvous
+##### Commission a device configured to bypass Rendezvous
 
-The command below pair a device with the provided IP address and port of the
-server to talk to.
+The command below commissions a device with the provided IP address and port of
+the server to talk to.
 
     $ chip-tool pairing bypass 192.168.0.30 5540
 
-#### Pair a device over BLE
+#### Commission a device over BLE
 
 Run the built executable and pass it the discriminator and pairing code of the
-remote device.
+remote device, as well as the network credentials to use.
 
 The command below uses the default values hard-coded into the debug versions of
-the ESP32 all-clusters-app:
+the ESP32 all-clusters-app to commission it onto a Wi-Fi network:
+
+    $ chip-tool pairing ble-wifi ssid password 0 20202021 3840
```

Comment:
following your PR , then someone must change the internal wording of chip-tool itself to commission

Reply:
I think that would be a good idea, yes, but didn't want to scope-creep this PR...